pytentiostat/tester.py

```diff
@@ -85,12 +102,25 @@ def read_write(
         Conversion factor to correct the current and voltage readings
     sr: float
         Shunt resistor used to correct the current reading
-    data: dict
+    config_data: dict
         Dictionary that contains the data read from the config file.
+    times: list
+        Contains the time measured during the experiment at each data point
+    voltages: list
+        Contains the voltage measured at each data point
+    currents: list
+        Contains the current measured at each data point
 
     Returns
     _______
-    nothing
+    times: list
+        Contains the time measured during the experiment at each data point

```

Comment:
This seems to be returning the same thing that is passed to the function based on the description.  Can you make it clearer what is passed in and what is returned?